### PR TITLE
Astolfo picture command

### PR DESCRIPTION
**Pull request by Computerfile**
Added command that sends picture of Astolfo, I had to parse JSON in Java I wanna die.

Also, Agent requested this idk why:

I do not love you except because I love you;
I go from loving to not loving you,
From waiting to not waiting for you
My heart moves from cold to fire.

I love you only because it's you the one I love;
I hate you deeply, and hating you
Bend to you, and the measure of my changing love for you
Is that I do not see you but love you blindly.

Maybe January light will consume
My heart with its cruel
Ray, stealing my key to true calm.

In this part of the story I am the one who
Dies, the only one, and I will die of love because I love you,
Because I love you, Love, in fire and blood. 

-Pablo Neruda

### CODE REVIEW

**Comment by AstolfoDev** [comment]
Again, is it at all possible to see any test runs of the code and ideally could I see the code in action (you running the command and the bot sending you the messages on Discord).
Cheers, mate!

File: src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                Scanner sc = new Scanner(url.openStream());
+
+                while(sc.hasNext()) {
+                    inline += sc.nextLine();
+                }
+                sc.close();
+
+                String image_link = "";
+                while(true){
+                    JsonElement jelement = new JsonParser().parse(inline);
```
Comment (AstolfoDev) [inline src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java:46]: This is deprecated, please update your methods to the latest methods in the library.

**Review by AstolfoDev — CHANGES_REQUESTED** [review]
Update deprecated methods from the dependencies to their latest instalments.

**Comment by Computerfile** [comment]
Here's the proof you requested

Untitled

File: src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java (hunk 1 above)
Comment (Computerfile) [reply src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java:46]: Unfortunately, of my knowledge, there aren't any good JSON parsing libraries in Java. The one I used is the best one that I know of

Comment (AstolfoDev) [reply src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java:46]: Yes, I'm aware GSON is the best library but you're using methods the library maintainers have marked as deprecated, please use the appropriate methods that the library developers intend for you to use.

**Comment by Computerfile** [comment]
Here's the proof you asked for

Untitled

File: src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java (hunk 1 above)
Comment (Znunu) [reply src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/fun/RandomAstolfo.java:46]: smh, imagine using deprecated methods. If the GSON maintainers see us doing this, they'll never forgive us

**Comment by AstolfoDev** [comment]
Closed due to inactivity.

**Comment by Znunu** [comment]
Oof

**Pull request closed without merging**